### ISSUE TITLE
References invalid column name when using Postgresql 12

### ISSUE THREAD

**Issue by filipeamoreira** [seamusabshere/upsert#130]
While testing Postgresql 12 in one of our apps we bumped into this issue:

```ruby
PG::UndefinedColumn: ERROR:  column d.adsrc does not exist
LINE 1: ...format_type(a.atttypid, a.atttypmod) AS sql_type, d.adsrc AS...
```
The column adsrc had a deprecation notice since v8.0 (https://www.postgresql.org/docs/8.0/catalog-pg-attrdef.html) and was removed in v12.0.

PR added here with fix: https://github.com/seamusabshere/upsert/pull/129

**Comment by itay-grudev** [comment]
@seamusabshere This impacts a lot of people right now, Please update it ASAP,

**Comment by pnomolos** [comment]
Fixed via #129
